deltastream-msk-vpc-endpoint-service/README.md

```diff
@@ -0,0 +1,70 @@
+# deltastream-msk-vpc-endpoint-service
+
+Terraform module that configures a AWS Private Link endpoint for an existing Amazon MSK cluster for connecting to DeltaStream.
+
```

Comment:
Add a section  `## install steps`  listing the terraform commands to run in order will be helpful:
```
# create tfvars file with your information, see vpces.tfvars example in examples/ folder
terraform init
terraform plan
terraform apply
```